Python/sysmodule.c

```diff
@@ -1994,6 +1994,58 @@ sys_getandroidapilevel_impl(PyObject *module)
 }
 #endif   /* ANDROID_API_LEVEL */
 
+/*[clinic input]
+sys.activate_perf_trampoline
```

Comment:
Is "perf_trampoline" really the public-facing name we want to give this? I think it's fine for the C APIs, though I note the xoption is abbreviated to just "perf".

Would something like "native_traceability" or "performance_tracing_mode" be more appropriate for the Python APIs?

Reply:
I am quite bad at names so I will accept any suggestions? Any objections to Steve's proposal? @tiran @gpshead ?

Reply:
I think we should name it after what it actually does. `activate_python_c_stack_frames` or `activate_stack_trampoline` perhaps.  agreed that we shouldn't put one specific platforms tool in the API name itself.

`sys.activate_stack_trampoline("perf")` makes a lot of sense as an API.  _(that'd be a required parameter and the only allowed value today)_  Deactivate does not need a parameter.

related to this: the `-Xperf` command line flag would be better off as `-Xstack_trampoline=perf`. As that carries the feature name lining up with the `sys` API and the specific implementation. Assume mac and windows and or alternates in the long term would add their own =frob values in the future.  (no need parse the = for values just string match on "stack_trampoline=perf" today)